galaxy_ng/app/tasks/synclist.py

```diff
@@ -155,4 +163,5 @@ def curate_synclist_repository(synclist_pk, **kwargs):
     else:
         raise RuntimeError("Unexpected synclist policy {}".format(synclist.policy))
 
+    log.debug('task_kwargs: %s', pprint.pformat(task_kwargs))
```

Comment:
do we pprint our log output? `pformat` generates indentation and I think we need to keep log (when output is a file) in to single lines and let formatting to the handlers and/or reader.